stable/mongodb-replicaset/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mongodb-replicaset
 home: https://github.com/mongodb/mongo
-version: 3.11.5
+version: 3.12.5
```

Comment:
3.12.0

Reply:
@steven-sheehy I've updated the version too. Thanks